services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/docker_compose_utils.py

```diff
@@ -67,7 +67,7 @@ async def docker_compose_config(
     """
     result = await _write_file_and_spawn_process(
         compose_spec_yaml,
-        command='docker-compose --file "{file_path}" config',
+        command='docker-compose --verbose --file "{file_path}" config',
```

Comment:
what about adding it as argument to the function?

```python

async def docker_compose_...(compose_spec_yaml: str, *, timeout: int=60, verbose: bool=False) -> CommandResult:
  options = []
  if verbose:
      options.append("--verbose")

  # ...
  command= f'docker-compose {" ".join(options)} --file "{{file_path}}" config',

```

This way this slowly converges into a meaningful API for docker-compose commands

Reply:
ok I did a variation around this and @GitHK request.